chart/templates/mayastor/apis/api-rest-deployment.yaml

```diff
@@ -24,6 +24,9 @@ spec:
         {{- include "base_pull_secrets" . }}
       initContainers:
         {{- include "base_init_containers" . }}
+      {{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
already defined below

Reply:
Will remove the one below as it's hardcoded